first_app/api/post.py

```diff
@@ -0,0 +1,114 @@
+import datetime
+from first_app.db import db
+from first_app.config import SECRET_KEY
+from .login_required import login_required
+from first_app.models import Post, FileUpload
+from flask import Flask, g, request, render_template, flash, Blueprint
+
+post_urls = Blueprint("psot",__name__)
+
```

Comment:
Also, add api for fetch single post with comments. like /api/v1/post/<post_id> -> GET